README.md

```diff
@@ -46,66 +47,77 @@ it("is a basic test with the page", async ({ page }) => {
 The test runner provides browser primitives as arguments to your test functions. Test functions can use one or more of these arguments.
 
 - `page`: Instance of [Page][page]. Each test gets a new isolated page to run the test.
-- `context`: Instance of [BrowserContext][browser-context]. Each test gets a new isolated context to run the test. The `page` object belongs to this context.
-  - `contextOptions`: Default options passed to context creation. Learn [how to modify them](#modify-context-options).
-- `browser`: Instance of [Browser][browser]. Browsers are shared across tests to optimize resources. Each worker process gets a browser instance.
-  - `browserOptions`: Default options passed to browser/launch creation.
+- `context`: Instance of [BrowserContext][browser-context]. Each test gets a new isolated context to run the test. The `page` object belongs to this context. Learn [how to configure](#modify-options) context creation.
+- `browser`: Instance of [Browser][browser]. Browsers are shared across tests to optimize resources. Each worker process gets a browser instance. Learn [how to configure](#modify-options) browser launch.
+- `browserName`: The name of the browser currently running the test. Either `chromium`, `firefox` or `webkit`.
 
 #### Specs and assertions
 
-- Use `it` and `describe` to write test functions. Run a single test with `it.only` and skip a test with `it.skip`.
+- Use `test` and `test.describe` to write test functions. Run a single test with `test.only` and skip a test with `test.skip`.
 - For assertions, use the [`expect` API](https://jestjs.io/docs/en/expect).
 
 ```js
-const { it, describe } = require("@playwright/test");
+const { test, describe } = require("@playwright/test");
 
-describe("feature foo", () => {
-  it("is working correctly", async ({ page }) => {
+test.describe("feature foo", () => {
+  test("is working correctly", async ({ page }) => {
     // Test function
   });
 });
 ```
 
+### Write a configuration file
```

Comment:
Let's get ourselves `npx playwright-test init` instead and move it up to where you create first test file.